src/content/products/deepsparse/community.mdx

```diff
@@ -202,7 +202,7 @@ The [GitHub repository](https://github.com/neuralmagic/deepsparse) includes pack
 
 ### Scheduling Single-Stream, Multi-Stream, and Elastic Inference
 
-The DeepSparse Engine offers up to three types of inferences based on your use case. Read more details here: [Inference Types](https://github.com/neuralmagic/deepsparse/blob/main/docs/source/scheduler.md).
+DeepSparse offers up to three types of inferences based on your use case. Read more details here: [Inference Types](https://github.com/neuralmagic/deepsparse/blob/main/docs/source/scheduler.md).
```

Comment:
```suggestion
DeepSparse offers up to three types of inferences based on your use case. Read [more about inference types](https://github.com/neuralmagic/deepsparse/blob/main/docs/source/scheduler.md).
```

Reply:
Stylistically, avoid using the word filler "here" and tell the user where/what you want them to do really do.